### DISCUSSION TITLE
Welcome to vimix Discussions!

### DISCUSSION THREAD

**Discussion by brunoherbelin** [brunoherbelin/vimix#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the vimix community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and suggestions.
  * Engage with other community members.

The Wiki pages provide some documentation but Q&A in Discussions can also be used to get to learn vimix.
The Issues page is best for reporting bugs ; share your Ideas in Discussions to request features or make suggestions.

**Comment by mateuszrealizator29** [comment]
When Windows release will be available?
